interval/lift/enrichment.go

```diff
@@ -130,28 +149,41 @@ func EnrichmentPValueUpperBound(elements1 []Lift, elements2 []Lift, noGapRegions
 		log.Println("Calculating the pValue.")
 	}
 
-	pValue, underflow := numbers.BinomialDist(numTrials, overlapCount, prob)
-	for s := overlapCount + 1; s <= numTrials && !(underflow && float64(s) > float64(numTrials)*prob); s++ {
+	enrichPValue, underflow = numbers.BinomialDist(numTrials, overlapCount, prob)
+	for s = overlapCount + 1; s <= numTrials && !(underflow && float64(s) > float64(numTrials)*prob); s++ {
```

Comment:
Should it be !(underflow && float64(s) > float64(numTrials)*prob) here or (!underflow && float64(s) > float64(numTrials)*prob) like line 159 below?